ibis/pyspark/tests/test_timecontext.py

```diff
@@ -13,3 +15,37 @@ def test_table_with_timecontext(client):
     expected = table.compile().toPandas()
     expected = expected[expected.time.between(*context)]
     tm.assert_frame_equal(result, expected)
+
+
+def test_time_indexed_window(client):
+    table = client.table('time_indexed_table')
+    window1 = ibis.trailing_window(
```

Comment:
Can you add tests for future window as well

Reply:
Sure, I add one to test `following` and compare the result with df generated by pyspark window.